Libraries/Opc.Ua.Server/Subscription/MonitoredItem.cs

```diff
@@ -1079,6 +1079,92 @@ public virtual void QueueEvent(EventFieldList fields)
             }
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="context"></param>
+        /// <param name="filter"></param>
+        /// <param name="instance"></param>
+        /// <returns></returns>
+        protected bool CanSendFilteredAlarm(FilterContext context, EventFilter filter, IFilterTarget instance)
```

Comment:
Not shure about the naming of the method. Maybe more like ValidateAgainstEventFilter (not shure about that either)?

Reply:
Name is consistent with operation.  Validation would not be a proper name for the operation.